layouts/partials/footer.html

```diff
@@ -121,8 +121,7 @@ <h3 class="h6 text-white">Comparisons</h3>
         <ul class="list-group list-group-transparent list-group-white list-group-flush list-group-borderless mb-0">
           <li><a class="list-group-item list-group-item-action" href="/blog/email-alias-vs-plus-sign/">vs Plus Sign (+) Trick </a></li>
           <li><a class="list-group-item list-group-item-action" href="/blog/vs-firefox-relay/">vs Firefox Relay</a></li>
-          <li><a class="list-group-item list-group-item-action" href="/blog/vs-burner-mail/">vs Burner Mail </a></li>
-          <li><a class="list-group-item list-group-item-action" href="/blog/alternative-33mail/">vs 33mail </a></li>
```

Comment:
You wanted to remove these two?

Reply:
Yes! Pages not performing so I'd like to remove them to make space to add other pages